docs/app/cloud/devspace/sharing_devspaces.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Sharing Devspace
+sidebar_position: 4
+hide_title: false
+---
+
+**Jetify Projects** are a great way to share configuration and secrets with your team. A project can store repository settings and secrets that are shared across all Devspace users in the project. For example, you can configure a project for a Backend API with the database parameters, API keys, and other secrets. When a user launches Jetify Devspace in the project, they will automatically have access to the project's configuration and secrets.
+
+## Creating a Project
+
+To create a project:
+
+1. Navigate to the **Projects** tab in the Jetify Dashboard
+2. Click on the `Create New` button on the top right corner of the page
+3. In the modal that appears, give the project a name, and then click "Create Project"
+4. In the new project, navigate to the Settings tab, and then click "Connect with Github" to connect the project to a Github repository
```

Comment:
No longer need to navigate to the Settings tab unless you have created a sandbox for that project already (the empty state also has a `Connect with Github` or `Choose Repository`